standalone/content/js/standaloneRoomViews.jsx

```diff
@@ -788,6 +802,84 @@ loop.standaloneRoomViews = (function(mozL10n) {
     }
   });
 
+  var Slide = React.createClass({
+    propTypes: {
+      imageClass: React.PropTypes.string.isRequired,
+      indexClass: React.PropTypes.string.isRequired,
+      joinRoom: React.PropTypes.func.isRequired,
+      text: React.PropTypes.string.isRequired,
+      title: React.PropTypes.string.isRequired
+    },
+
+    handleGotItClick: function(event) {
```

Comment:
In addition to spinning off a new bug for this, please add XXX comments around the relevant pieces that will be called back so that future code readers will get relevant ideas about what to do with the code.